site/content/docs/5.2/utilities/background.md

```diff
@@ -14,6 +14,7 @@ Similar to the contextual text color classes, set the background of an element t
 {{< colors.inline >}}
 {{- range (index $.Site.Data "theme-colors") }}
 <div class="p-3 mb-2 bg-{{ .name }}{{ if .contrast_color }} text-{{ .contrast_color }}{{ else }} text-white{{ end }}">.bg-{{ .name }}</div>
+<div class="p-3 mb-2 bg-{{ .name }}-subtle text-emphasis-{{ .name }}">.bg-{{ .name }}-subtle</div>
 {{- end -}}
 {{< /colors.inline >}}
 <div class="p-3 mb-2 bg-body text-dark">.bg-body</div>
```

Comment:
There's an issue here with `.bg-body` where we can't see the text in dark mode:
![Screenshot 2022-11-18 at 18 19 54](https://user-images.githubusercontent.com/17381666/202764234-4b66eab7-3a46-4fc8-851c-ea7bae6db9d7.png)